keras_nlp/applications/bert.py

```diff
@@ -11,22 +11,23 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-"""Bert model and layer implementations.
-
-These components come from the tensorflow official model repository for BERT:
-https://github.com/tensorflow/models/tree/master/official/nlp/modeling
-
-This is to get us into a testable state. We should work to replace all of these
-components with components from the keras-nlp library.
-"""
+"""Bert model and layer implementations."""
 
 import tensorflow as tf
 from tensorflow import keras
 
-import keras_nlp
+import keras_nlp.layers
+
+# isort: off
```

Comment:
can we delete these whole three lines? I don't think we need them

Reply:
Yes heard from `keras_cv` folks that we're relying on `init` for now.